hexa/pipelines/schema/mutations.py

```diff
@@ -185,7 +185,7 @@ def resolve_run_pipeline(_, info, **kwargs):
             user=request.user,
             pipeline_version=version,
             trigger_mode=PipelineRunTrigger.MANUAL,
-            config=input.get("config", {}),
+            config=input.get("config", version.config),
```

Comment:
No need to use `version.config` as the default for the config. This logic has to be in the `run` method of the `PipelineVersion` in itself.

Reply:
Ok, changed it to merge both